src/main/java/com/fasterxml/jackson/databind/deser/std/StdDeserializer.java

```diff
@@ -596,7 +596,8 @@ protected final byte _parseBytePrimitive(JsonParser p, DeserializationContext ct
         CoercionAction act = _checkFromStringCoercion(ctxt, text,
                 LogicalType.Integer, Byte.TYPE);
         if (act == CoercionAction.AsNull) {
-            return (byte) 0; // no need to check as does not come from `null`, explicit coercion
```

Comment:
@cowtowncoder could you explain why you added this comment before?

Reply:
I do not remember the context here, but guessing it might have come with copy-paste.